test/hybig/HyBIG_Regression.ipynb

```diff
@@ -1109,17 +819,17 @@
     "            )\n",
     "        ]\n",
     "\n",
-    "        # Perform the same checks for all tiles:\n",
-    "        for tile_basename, expected_tile_metadata in prefire_tile_information:\n",
+    "        for tile_basename in prefire_tiles:\n",
     "            tile_files = build_file_list(tile_basename, colour_map_reference_dir, 'PNG')\n",
     "            tile_test_files = build_file_list(tile_basename, Path(temp_dir), 'PNG')\n",
```

Comment:
Can you rename these tile_files -> tile_reference_files
tile_test_files -> tile_result_files?

It's in both tiled tests. tiled output and colour test

Reply:
Good call. [c38cd6](https://github.com/nasa/harmony-regression-tests/pull/105/commits/c38cd6638ccbf8bc7c684433cdde0426ac2bf763)

(I also adopted `test_result_files` in the non-tiled tests)